app/models/work.rb

```diff
@@ -0,0 +1,27 @@
+class Work < ApplicationRecord
+  has_many :votes
+
+  validates_presence_of :title, :category
+
+  validates :title, uniqueness: { scope: :category, message: "Cannot have the same title within the same media type" }
+
+  def vote_count
+    self.votes.count
+  end
+
+  def self.ordered_by_votes_works(category)
+    works = Work.all.where(category: category).to_a
```

Comment:
`Work.all.where(category: category)` actually returns an array, so I don't think you need to call `.to_a` on this to turn it into an array! :O